integration/query_fuzz_test.go

```diff
@@ -162,9 +146,18 @@ func TestDisableChunkTrimmingFuzz(t *testing.T) {
 	queryEnd := time.Now().Add(-time.Minute * 20)
 	cases := make([]*testCase, 0, 200)
 	testRun := 500
+	var (
+		expr  parser.Expr
+		query string
+	)
 	for i := 0; i < testRun; i++ {
-		expr := ps.WalkRangeQuery()
-		query := expr.Pretty(0)
+		for {
+			expr = ps.WalkRangeQuery()
+			query = expr.Pretty(0)
+			if !strings.Contains(query, "timestamp") {
```

Comment:
We on purposely ignore `timestamp` to reduce flakiness as it is a know imcompatible behavior